src/ide/completion/helpers/item.rs

```diff
@@ -28,8 +28,9 @@ pub enum CompletionRelevance {
     Lowest = 0,
     Low = 1,
     Medium = 2,
-    High = 3,
-    Highest = 4,
+    MediumHigh = 3,
```

Comment:
Let's make those names actually mean something semantically (associate it with some item types etc. or anything that actually leads us to conclusion where should we place the item in the list).